Dockerfile.local_cdk

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9-slim
```

Comment:
A single Dockerfile like this one may be problematic if it assumes what connectors' Dockerfiles look like (the dockerfile may deviate from the "vanilla" ones). Though TBH my guess is that 95% of the time it's not an issue? 

in any case, is our life easier if we do something a _little_ boneheaded (but effective) and do something like run a shell script with code like: 

```
cp airbyte-cdk/python airbyte-integrations/connectors/$CONNECTOR/build/airbyte-cdk
cp airbyte-integrations/connectors/$CONNECTOR/Dockerfile airbyte-integrations/connectors/$CONNECTOR/build/Dockerfile.dev # Copy the connector's Dockerfile into build/Dockerfile.dev
cd airbyte-integrations/connectors/$CONNECTOR/
echo "RUN pip install -e ./build/airbyte-cdk" >> build/Dockerfile.dev # install local CDK
docker build . -f build/Dockerfile.dev -t <tag> 
cd - 
```

assuming we `.gitignore` build, it'd achieve something similar to what this is doing without polluting anything meaningful

Reply:
This would also allow us to avoid having to `sed` stuff and fiddle with site-packages